### ISSUE TITLE
Issues accessing config page (+app?) when using yarn serve

### ISSUE THREAD

**Issue by Forceh91** [Forceh91/retro-env-can-weather-chan#316]
Opening a new issue based off a comment left by @fatwreck in #306 

> However, I'm unable to access the config page but it could be me. The README often references `http://localhost:8600/`, but in my understanding, the typical default is `http://localhost:8080/`, so I'm not sure if this is a Vue quark or the package.json should set port to 8600 to get everyone on the same page. If it's the latter, I think changing line 6 on the `package.json` to `vue-cli-service serve --port 8600`
> 
> My local is using `http://localhost:8080/` but hitting `http://localhost:8080/#/config/` gets stuck on `Fetching config... please wait.` with a console error `http://localhost:8080/api/init 404 (Not Found)`
> 
> The `req` argument in the `startBackend` function in the `backend.js` file is empty/unused. Maybe it's populated from the missing(?) `api/init` file? Sounds like a file, but an init file would be unusual. idk :)

**Comment by Forceh91** [comment]
@fatwreck I think this solves your issue, feel free to open the ticket again if not.

**Comment by fatwreck** [comment]
Thanks for the updates.
I may be running the project incorrectly as it's still not compiling.

I'm on the master branch, pulled latest, using node v14.20, yarn install, yarn serve.

When I run `yarn build`, it creates a build folder for deploy which is fine, but it sounds like it's also expected to serve the site as well. That's a new one for me. I thought `yarn serve` is to run, and `yarn build`'s only functionality is to make a build folder.

Clicked Clear Site Data in the Application tab and did a hard refresh to see if it was cacheing.

In the network tab, error on init. 
**Header Tab**
`Request URL: http://localhost:8080/api/init`
`Request Method: GET`
`Status Code: 500 Internal Server Error`
`Remote Address: 127.0.0.1:8080`

**Terminal Output:**
` /Web/retro-env-can-weather-chan > master > yarn serve`
`yarn run v1.22.18`
`$ vue-cli-service serve`
`Browserslist: caniuse-lite is outdated. Please run:`
`npx browserslist@latest --update-db`
`Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating`
`INFO  Starting development server...`
`98% after emitting CopyPlugin`

` DONE  Compiled successfully in 4032ms                                                                                                4:00:42 PM`


`  App running at:`
`  - Local:   http://localhost:8080/ `
`  - Network: http://192.168.0.247:8080/`

`  Note that the development build is not optimized.`
`  To create a production build, run yarn build.`

`Proxy error: Could not proxy request /api/init from localhost:8080 to http://localhost:8600.`
`See https://nodejs.org/api/errors.html#errors_common_system_errors for more information (ECONNREFUSED).`

**Comment by Forceh91** [comment]
You need to make sure that you have done `node backend.js` in a separate terminal window too, that's where the data comes from.

You only need to do `yarn build` or `yarn serve` not both. If you are not developing then I recommend using `yarn build` and then just using `node backend.js`. Once that is done then you should be able to access the channel from http://localhost:8600.

**Comment by fatwreck** [comment]
Ah, there's the disconnect. Thanks so much for clarifying!

I was misunderstanding the ReadMe thinking that you configured first, then ran the backend.js script. Makes perfect sense the other way around.

**Comment by Forceh91** [comment]
Ah that is fully my bad, I didn't properly update the README after adding the config page. I've just reworked it so it should make more sense now 😅
